queries/move-to-LoketLB-graphs.sparql

```diff
@@ -0,0 +1,216 @@
+PREFIX besluit: <http://data.vlaanderen.be/ns/besluit#>
+PREFIX ext: <http://mu.semte.ch/vocabularies/ext/>
+PREFIX prov: <http://www.w3.org/ns/prov#>
+PREFIX mu: <http://mu.semte.ch/vocabularies/core/>
+PREFIX skos: <http://www.w3.org/2004/02/skos/core#>
+PREFIX rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#>
+PREFIX mandaat: <http://data.vlaanderen.be/ns/mandaat#>
+PREFIX org: <http://www.w3.org/ns/org#>
+PREFIX lblodlg: <http://data.lblod.info/vocabularies/leidinggevenden/>
+PREFIX dct: <http://purl.org/dc/terms/>
+PREFIX extlmb: <http://mu.semte.ch/vocabularies/ext/lmb/>
+PREFIX schema: <http://schema.org/>
+PREFIX person: <http://www.w3.org/ns/person#>
+PREFIX temp: <http://mu.semte.ch/graph/temp/>
+
+# Inserting bestuurseenheid
+# Should stay in public graph
+# INSERT {
+#   GRAPH ?targetGraph {
+#     ?bestuurseenheid ?p ?o.
+#     ?os ?op ?bestuurseenheid.
+#   }
+# } WHERE {
+#   GRAPH <http://mu.semte.ch/graphs/public> {
+#     ?bestuurseenheid a besluit:Bestuurseenheid;
+#                      mu:uuid ?id.
+#     {
+#       # Outgoing
+#       ?bestuurseenheid ?p ?o.
+#       VALUES ?p { rdf:type mu:uuid skos:prefLabel skos:altLabel ext:wilMailOntvangen ext:mailAdresVoorNotificaties ext:isTrailUser ext:viewOnlyModules
+#                   besluit:werkingsgebied ext:inProvincie besluit:classificatie }
+#     }
+#     UNION
+#     {
+#       # Incoming
+#       ?os ?op ?bestuurseenheid.
+#       VALUES ?op { ext:contactVoor besluit:bestuurt ext:origineleBestuurseenheid }
+#     }
+#   }
+#   BIND(IRI(CONCAT("http://mu.semte.ch/graphs/organizations/", ?id, "/LoketLB-mandaatGebruiker")) AS ?targetGraph)
+# };
+# Inserting bestuurseenheid-contact
+# bestuurseenheid-contact doesn't exit in the loket data
+
+# Inserting werkingsgebied
+# Should stay in public graph
+# INSERT {
+#   GRAPH ?targetGraph {
+#     ?werkingsgebied ?p ?o.
+#   }
+# } WHERE {
+#   GRAPH ?targetGraph {
+#     ?bestuurseenheid besluit:werkingsgebied ?werkingsgebied.
+#   }
+#   GRAPH <http://mu.semte.ch/graphs/public> {
+#     ?werkingsgebied a prov:Location;
+#                     ?p ?o.
+#     VALUES ?p { rdf:type mu:uuid rdfs:label ext:werkingsgebiedNiveau }
+#   }
+# };
+# UNDER CONSTRUCTION, IN THE PROCESS OF ADDING ext:inGraph
+# Inserting bestuursorgaan
+INSERT {
+  GRAPH ?targetGraph {
+    ?orgaan ?p ?o.
+    ?fakeOrgaan ?pp ?oo.
+    ?os ?op ?orgaan.
+    ?oss ?opp ?fakeOrgaan.
+    ?orgaan ext:inGraph ?targetGraph.
+    ?fakeOrgaan ext:inGraph ?targetGraph.
+  }
+} WHERE {
+  GRAPH <http://mu.semte.ch/graphs/public> {
+    ?bestuurseenheid a besluit:Bestuurseenheid;
+                     mu:uuid ?id;
+                     ^besluit:bestuurt ?orgaan;
+                     ^ext:origineleBestuurseenheid ?fakeOrgaan.
+    ?orgaan a besluit:Bestuursorgaan.
+    ?fakeOrgaan a besluit:Bestuursorgaan.
+    {
+      ?orgaan ?p ?o.
+      VALUES ?p { rdf:type mu:uuid skos:prefLabel ext:deactivatedAt
```

Comment:
I would not care about this, you're making things harder than they need to be. Just take all predicates with you.

Normally, you don't need to move the predicates in the opposite direction as they will be moved with the subject of the type of the origin of the predicate, but it might be good to double-check

Reply:
I specify the predicates because then you are aware of what data you are holding and transitioning. But if all data can be transferred, then it isn´t necessary.

Also specifying the inverse triples is uselfull in order to "pull" the next type into the correct graph. With the new ext:inGraph approach, this might not be necessary anymore.